src-lib/data.cpp

```diff
@@ -128,7 +112,7 @@ matrix load_image_paths_gray(char **paths, int n, int w, int h)
     X.cols = 0;
 
     for(i = 0; i < n; ++i){
-        image im = load_image(paths[i], w, h, 3);
+        image im = load_image(paths[i], w, h, 3);	// #COLOR
```

Comment:
I assume you're leaving this for you to remember to go back and fix up the "3"?  Add a Doxygen todo:  `///< @todo #COLOR`